pkg/karmadactl/cmdinit/kubernetes/deployments.go

```diff
@@ -558,7 +558,6 @@ func (i *CommandInitOption) makeKarmadaControllerManagerDeployment() *appsv1.Dep
 				Operator: corev1.TolerationOpExists,
 			},
 		},
-		ServiceAccountName:           controllerManagerDeploymentAndServiceName,
```

Comment:
Why remove this by this PR?

Reply:
> By the way, are you going to update the version for the chart? I'm asking because I see this patch including both development and karmadactl, just the chart left. I'm ok to do it by separated PR, just make sure the chart part is still on your plan.

On the way

Reply:
in this PR, will rebase after it merged: https://github.com/karmada-io/karmada/pull/2547